Sources/App/Views/PackageController/PackageShow+Model.swift

```diff
@@ -232,6 +231,18 @@ extension PackageShow.Model {
         )
     }
     
+    func archivedListItem() -> Node<HTML.ListContext> {
+         if isArchived {
+             return .li(
+                 .class("archived"),
+                 .strong("No longer in active development.")
```

Comment:
I know I said in #1174 that the proposed wording was good, but now I look at it in place, I was wrong. 😬

We should move more of the text from the second line up into this item. So it should say:

> **No longer in active development.** The package author has archived this project and the repository is read-only.

Then, the `history` item should say:

> It had XXX commits and XXX releases before being archived.

Reply:
Actually, now that `history` line reads strangely as it's not super clear what `It` refers to where it was when it followed on after the previous sentence. How about this for the `history` line:

> Before being archived, it had XXX commits and XXX releases.